src/erc7432/role/grant-handler.ts

```diff
@@ -1,55 +1,48 @@
 import { log } from '@graphprotocol/graph-ts'
-import { RoleGranted } from '../../../generated/Roles/ERC7432Roles'
+import { RoleGranted } from '../../../generated/ERC7432-Immutable-Roles/ERC7432'
 import { Account, Nft, Role } from '../../../generated/schema'
-import { generateNftId, generateRoleId } from '../../utils/helper'
+import { generateNftId, generateRoleId, findOrCreateAccount } from '../../utils/helper'
 
 export function handleRoleGranted(event: RoleGranted): void {
   const tokenId = event.params._tokenId.toString()
-  const tokenAddress = event.params._tokenAddress.toHexString()
+  const tokenAddress = event.params._tokenAddress.toHex()
 
-  const nftId = generateNftId(tokenId, tokenAddress)
+  const nftId = generateNftId(tokenAddress, tokenId)
   const nft = Nft.load(nftId)
-
   if (!nft) {
     log.warning('[handleRoleGranted] NFT {} does not exist, skipping...', [nftId])
     return
   }
 
-  const address = event.transaction.from.toHexString()
-  const grantor = Account.load(address)
-
+  const grantorAddress = event.params._grantor.toHex().toLowerCase()
+  const grantor = Account.load(grantorAddress)
```

Comment:
Maybe also call this `grantorAccount` to match `granteeAccount` var naming? 
```suggestion
  const grantorAccount = Account.load(grantorAddress)
```